packages/core/src/animation/Animator.ts

```diff
@@ -80,7 +80,7 @@ export class Animator extends Component {
    */
   play(stateName: string, layerIndex: number = -1, normalizedTimeOffset: number = 0): void {
     if (this._controllerUpdateFlag?.flag) {
-      this._clearPlayData();
+      this._reset();
```

Comment:
I found `_reset` also have rervert defalut value, the same with `_revertDefaultValue`,may be the revert logic should optimization